packages/runtime/src/helpers/functions.ts

```diff
@@ -35,6 +35,11 @@ export const generateFunctions = async (
   const publishDir = relative(functionDir, publish)
 
   for (const { route, config, compiled } of apiRoutes) {
+    if (config.type === undefined) {
```

Comment:
Just testing this out first, but I may go to where `apiRoutes` is created and filter there.